internal/controller/stages/stages.go

```diff
@@ -682,7 +696,9 @@ func (r *reconciler) syncControlFlowStage(
 func (r *reconciler) syncNormalStage(
 	ctx context.Context,
 	stage *kargoapi.Stage,
+	nowFn func() time.Time,
```

Comment:
Same as [here](https://github.com/akuity/kargo/pull/1815/files#r1563362958)

Reply:
Resolved in https://github.com/akuity/kargo/pull/1815/commits/fff47409b15ecc2dd8ebed60443f9b17cd8b4aae